src/index.html

```diff
@@ -49,5 +49,7 @@ <h2 class="font-semibold text-2xl">Add your score</h2>
         </div>
       </div>
     </main>
+
+    <script src="index.js"></script>
```

Comment:
[OPTIONAL] When working with Webpack, there is no need to add JS script tags to our index.html template in most cases. That's because Webpack automatically bundles our JS for production and includes it on the head of our file. Including it by ourselves may lead to unexpected bugs.

Reply:
Oooh okay. Great.